include/Core/Base/BaseModule.php

```diff
@@ -514,4 +514,11 @@ protected function enable_ajax_if_needed_to_dropdown_class_name( $count, $class_
 	protected function get_cron_label() {
 		return $this->messages['cron_label'];
 	}
+
```

Comment:
I have asked this function to be moved to the `Renderer` class.

This is what is specified in the original issue.

> If the helper function is not accessible in MetasModule, then move the render_post_type_dropdown function from PostModule to Renderer class which is the parent class of all these classes.